src/openforms/registrations/contrib/zgw_apis/tests/test_validators.py

```diff
@@ -0,0 +1,84 @@
+from django.test import TestCase
+
+import requests_mock
+from zds_client.oas import schema_fetcher
+from zgw_consumers.test import generate_oas_component
+from zgw_consumers.test.schema_mock import mock_service_oas_get
+
+from ..plugin import ZaakOptionsSerializer
+from .factories import ZgwConfigFactory
+
+
+@requests_mock.Mocker(real_http=False)
```

Comment:
```suggestion
@requests_mock.Mocker()
```
`real_http=False` is the default